sdk/identity/azure-identity/azure/identity/_credentials/environment.py

```diff
@@ -22,7 +22,23 @@
     EnvironmentCredentialTypes = Union["CertificateCredential", "ClientSecretCredential", "UsernamePasswordCredential"]
 
 
-class EnvironmentCredential:
+def get_credential_unavailable_message():
+    # type: () -> str
+    message = "Incomplete environment configuration. See documentation for expected environment variables"
```

Comment:
if there is a stable enough link that could be included here that would make the message more immediately actionable .

Reply:
Thanks, that's a good idea, I'll take it.